src/system/SystemPacketBuffer.h

```diff
@@ -592,7 +617,13 @@ class DLL_EXPORT PacketBufferHandle
      *
      * @note This should be used ONLY by low-level code interfacing with LwIP.
      */
-    struct pbuf * GetLwIPpbuf() { return static_cast<struct pbuf *>(mBuffer); }
+    struct pbuf * GetLwIPpbuf() {
+        if (mBuffer)
+        {
+            PacketBuffer::Check(mBuffer, __FILE__, __LINE__, __func__);
```

Comment:
This leaks filenames, function names and line numbers, bloating the resulting binary. Suggest wrapping `PacketBuffer::Check(mBuffer, __FILE__, __LINE__, __func__)` in a macro that uses "fixed" file/line/func if a given "check location enable constant" is not enabled, otherwise even if NDEBUG is set, you leak __FILE__ and __LINE__ directly.

Reply:
This PR isn't intended to leave draft stage, and has leftovers from local debugging.  I'm just trying to understand the github CI.